h2o-extensions/xgboost/src/main/java/hex/api/xgboost/RegisterRestApi.java

```diff
@@ -19,6 +20,42 @@ public void registerEndPoints(RestApiContext context) {
     XGBoost xgBoostMB = new XGBoost(true);
     // Register XGBoost model builder REST API
     registerModelBuilder(context, xgBoostMB, SchemaServer.getStableVersion());
+    // Register Remote XGBoost execution
```

Comment:
I gave this section half an hour of comments and after correcting myself for the 99999th time, I agree this is the best we can squeeze out of our API and I fully agree with the way it is done :wink: